src/modules/ethereum/chains/requests.ts

```diff
@@ -22,46 +21,22 @@ function fetchChains(
         scope: ['info'],
         payload,
       },
-      onData: ({ value }) => {
+      onData: ({ value, isStale }) => {
         if (value) {
           resolve(value);
-          unsubscribe();
+          if (!isStale) {
+            unsubscribe();
+          }
```

Comment:
Can you explain what's going on here? :)

Reply:
It is a way to avoid calling unsubscribe before initialisation.
As I understand, it can happen because of instant onData call with stale data. So I tried to avoid this case